2019-04-01T21:29:58.277+0000 I CONTROL  [initandlisten] MongoDB starting : pid=3978 port=27017 dbpath=/data/db 64-bit host=ip-172-31-18-151
2019-04-01T21:29:58.277+0000 I CONTROL  [initandlisten] db version v3.6.3
2019-04-01T21:29:58.277+0000 I CONTROL  [initandlisten] git version: 9586e557d54ef70f9ca4b43c26892cd55257e1a5
2019-04-01T21:29:58.277+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.1.0g  2 Nov 2017
2019-04-01T21:29:58.277+0000 I CONTROL  [initandlisten] allocator: tcmalloc
2019-04-01T21:29:58.277+0000 I CONTROL  [initandlisten] modules: none
2019-04-01T21:29:58.277+0000 I CONTROL  [initandlisten] build environment:
2019-04-01T21:29:58.277+0000 I CONTROL  [initandlisten]     distarch: x86_64
2019-04-01T21:29:58.277+0000 I CONTROL  [initandlisten]     target_arch: x86_64
2019-04-01T21:29:58.277+0000 I CONTROL  [initandlisten] options: { processManagement: { fork: true }, systemLog: { destination: "file", path: "/home/ubuntu/databaseTest/log/mongodb.log" } }
2019-04-01T21:29:58.278+0000 I STORAGE  [initandlisten] exception in initAndListen: DBPathInUse: Unable to lock the lock file: /data/db/mongod.lock (Resource temporarily unavailable). Another mongod instance is already running on the /data/db directory, terminating
2019-04-01T21:29:58.278+0000 I CONTROL  [initandlisten] now exiting
2019-04-01T21:29:58.278+0000 I CONTROL  [initandlisten] shutting down with code:100
